about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Global Summit</title>
+  <link rel="shortcut icon" href="img/CC-Global-Summit-2021.jpg" type="image/jpg">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/about.css">
+</head>
+<body>
+  <nav class="nav-1 bg-secondary">
+    <div class="container">
+      <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link" href="#"><i class="fa fa-facebook text-white"></i></a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#"><i class="fa fa-twitter text-white"></i></a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">My page</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="#">logout</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <nav class="navbar nav-menu navbar-expand-lg">
+    <button type="button" class="navbar-toggler button-nav border-0" data-toggle="collapse" data-bs-target="#navbarContent" aria-controls="navbarContent" aria-expanded="false">
+      <span class="navbar-toggler-icon" id="button-toggle"></span>
+    </button>
+  
+    <a class="navbar-brand img-brand" href="#"><img src="img/ruby-1.png" class="img-logo" alt=""></a>
+    <div class="collapse navbar-collapse pb-3 show" id="navbarContent">
+      <ul class="navbar-nav nav-list pe-0">
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-first" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link ms-4 fs-6 text-nav" href="#">Program</a>
```

Comment:
- [ ] Kindly on the about page change the link to the about page and make it to the Home page. so I can go back there.
![image](https://user-images.githubusercontent.com/35535130/181546571-9c7c4b12-9170-4f33-8a21-6c24098ff2eb.png)